asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -224,6 +235,31 @@ def to_oemol(self) -> oechem.OEMol:
         mol = _set_SD_data_repr(mol, data)
         return mol
 
+    def to_rdkit(self) -> "Chem.Mol":
```

Comment:
are we also copying over the ligand name here? we need to be careful to conserve this both ways as it's become clear that not having names associated with ligands is problematic downstream

Reply:
Yes we copy it over as an SD tag called `compound_name`, but we can add it as the `_Name` property as well.